ouroboros-consensus/src/Ouroboros/Consensus/Network/NodeToClient.hs

```diff
@@ -131,7 +138,7 @@ mkHandlers NodeKernelArgs {cfg, tracers} NodeKernel {getChainDB, getMempool} =
 
 -- | Node-to-client protocol codecs needed to run 'Handlers'.
 data Codecs' blk serialisedBlk e m bCS bTX bSQ = Codecs {
-      cChainSyncCodec    :: Codec (ChainSync serialisedBlk (Point blk) (Tip blk))  e m bCS
+      cChainSyncCodec    :: Codec (ChainSync (WithInterpreter blk serialisedBlk) (Point blk) (Tip blk))  e m bCS
```

Comment:
`WithInterpreterClient`?

Reply:
`Codecs'` is the general type (used by both client and server), so I think using the general `WithInterpreter` makes sense here. 

On another note `serialisedBlk` parameter is a bit misleading, since this can also be the blk.